policyengine_us/parameters/gov/states/dc/dhs/tanf/resource_limit/higher_limit.yaml

```diff
@@ -6,6 +6,10 @@ metadata:
   reference:
     - title: TANF for District Families
       href: https://dhs.dc.gov/service/tanf-district-families
+    - title: DC TANF State Plan 2024 | Section 4 (4)(a) 
+      href: https://dhs.dc.gov/sites/default/files/dc/sites/dhs/service_content/attachments/DC%20TANF%20State%20Plan_Oct-2023.pdf#page=40
```

Comment:
Lets include this legal code as well even if it does not state the value
https://code.dccouncil.gov/us/dc/council/code/sections/4-205.37